### PR TITLE
[test] add lang-expression plugin to vagrant tests

### PR DESCRIPTION
**Pull request by brwe**
we need this now that #13726 has been merged

### CODE REVIEW

File: qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash [hunk 1]
```diff
@@ -225,6 +225,10 @@ fi
     install_and_check_plugin store smb
 }
 
+@test "[$GROUP] install lang-expression plugin" {
+    install_and_check_plugin lang expression
```
Comment (nik9000) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: I've been putting these in alphabetical order.

File: qa/vagrant/pom.xml [hunk 2]
```diff
@@ -219,6 +219,12 @@
                                     <version>${elasticsearch.version}</version>
                                     <type>zip</type>
                                 </artifactItem>
+                                <artifactItem>
```
Comment (nik9000) [inline qa/vagrant/pom.xml]: I've been putting these in alphabetical order.

**Comment by nik9000** [comment]
Left minor comments otherwise LGTM.

**Comment by brwe** [comment]
ok, reordered

**Comment by rmuir** [comment]
I just backported this to 2.x

**Pull request merged**